src/GraphMetadataViewExtension/GraphMetadataView.xaml

```diff
@@ -0,0 +1,194 @@
+                <Setter Property="Foreground" 
+                        Value="{StaticResource NestedMemberTextColor}"/>
+                <Setter Property="Padding"
+                        Value="5" />
+            </Style>
+
+            <Style TargetType="{x:Type ListView}">
+                <Setter Property="Background"
+                        Value="Transparent" />
+                <Setter Property="BorderBrush"
+                    Value="Transparent" />
+                <Setter Property="Margin"
+                        Value="0" />
+            </Style>
+
+            <Style TargetType="{x:Type ListViewItem}">
+                <Setter Property="Template">
+                    <Setter.Value>
+                        <ControlTemplate TargetType="{x:Type ListViewItem}">
+                            <ContentPresenter />
+                        </ControlTemplate>
+                    </Setter.Value>
+                </Setter>
+            </Style>
+
+            <Style TargetType="{x:Type Button}">
+                <Style.Triggers>
+                    <Trigger Property="IsMouseOver"
+                         Value="True">
+                        <Setter Property="Cursor"
+                            Value="Hand" />
+                    </Trigger>
+                </Style.Triggers>
+            </Style>
+
+            <Style TargetType="{x:Type TextBlock}" x:Key="headerStyle">
+                <Setter Property="FontWeight"
+                        Value="Bold" />
+                <Setter Property="FontSize"
+                        Value="13" />
+                <Setter Property="Foreground"
+                        Value="{StaticResource NestedMemberTextColor}" />
+            </Style>
+
+            <Style TargetType="{x:Type GroupBox}"
+                   BasedOn="{StaticResource {x:Type GroupBox}}">
+                <Setter Property="BorderThickness"
+                        Value="0" />
+                <Setter Property="Margin"
+                        Value="0,5px" />
+                <Setter Property="Foreground"
+                        Value="{StaticResource NestedMemberTextColor}" />
+                <Setter Property="HeaderTemplate">
+                    <Setter.Value>
+                        <DataTemplate>
+                            <TextBlock Text="{Binding}"
+                                       Margin="0,0,0,5"
+                                       Style="{StaticResource headerStyle}" />
+
+                        </DataTemplate>
+                    </Setter.Value>
+                </Setter>
+            </Style>
+
+        </ResourceDictionary>
+    </UserControl.Resources>
+    
+    <Grid Margin="10px 10px">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <StackPanel Orientation="Vertical"
+                    Grid.Row="0">
+            <GroupBox Header="Description"
```

Comment:
I think this should start from a single line and be auto resizable like we have it in generative design tools